source/indexes/atlas-search-index.txt

```diff
@@ -20,12 +21,23 @@ Atlas Search Indexes
 Overview
 --------
 
-:atlas:`Atlas Search </atlas-search>` enables you to perform full-text searches on
+You can programmatically manage your :atlas:`Atlas Search
+</atlas-search>` and :atlas:`Atlas Vector Search
+</atlas-vector-search/vector-search-overview/>` indexes by using the
+{+driver-short+}.
+
+Atlas Search enables you to perform full-text searches on
 collections hosted on MongoDB Atlas. Atlas Search indexes specify the behavior of
 the search and which fields to index.
 
-You can call the following methods on a collection to manage your Atlas Search
-indexes:
+Atlas Vector Search enables you to perform semantic searches on vector
+embeddings stored in MongoDB Atlas. Vector Search indexes define the
+indexes for the vector embeddings that you want to query and the boolean,
+date, objectId, numeric, string, or UUID values that you want to use to
+pre-filter your data.
+
+You can call the following methods on a collection to manage your Atlas
+Search and Vector Search indexes:
 
 - ``createSearchIndex()``
 - ``createSearchIndexes()``
```

Comment:
Q: Does the note below (can't comment on it directly) apply just to Atlas Search indexes? If it applies to Vector search index methods as well, I'd update it to reflect that